openedx/core/lib/xblock_utils.py

```diff
@@ -396,3 +397,29 @@ def _course_info_content(html_parsed):
                         return payload
 
     return course_update_items
+
+
+def add_inline_analytics(user, block, view, frag, context):  # pylint: disable=unused-argument
```

Comment:
Which arg is unused? Can we prefix it with `_`? That should silence pylint (without needing the disable pragma), while also making it apparent which is being ignored.